src-cpp/app/boot/app_controller.h

```diff
@@ -29,7 +29,7 @@ class AppController : public QObject, AppControllerDelegate
     // Modules
     // To-Do make this a shared pointer and remove circular dependency.
     Modules::Startup::ModuleAccessInterface* m_startupModule;
-    std::unique_ptr<Modules::Main::IModuleAccess> m_mainModulePtr;
+    Modules::Main::IModuleAccess* m_mainModulePtr;
```

Comment:
minor: `m_mainModule` cos its not smart pointer anymore

Reply:
but it is still a pointer?

Reply:
it's convention for smart pointers - add `Ptr` to the end